src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -1114,6 +1114,19 @@
      * {@code LITERAL_FOR}.  The child of this element is an
      * optional expression list.
      *
+     * <p>For example:</p>
+     * <pre>
+     * for (int i = 0; i &lt; arr.length; i++) {}
+     * </pre>
+     * <p>parses as:</p>
+     * <pre>
+     * FOR_ITERATOR -&gt; FOR_ITERATOR
```

Comment:
Please print tree for whole FOR, user need to see context of where token is used

Reply:
done